### ISSUE TITLE
Export fails with AttributeError: 'NoneType' object has no attribute 'nodes'

### ISSUE THREAD

**Issue by Fak3** [kergalym/YABEE#26]
blender: 2.90
YABEE: git master

I tried to export this [file 
](https://drive.google.com/file/d/1IqbiYjHbL0ZTmlqRr3-0M2TLKLQT-PK5/view?usp=sharing) and got an error:

```
Objects for export: ['Cube', 'Light', 'Camera', 'Icosphere', 'Torus']
WRITE main EGG to /home/z/pproj/panda_tut/Models/pandatest1.egg
INFO: Used materials:  {'Material.002', 'Material', 'Material.001'}
Traceback (most recent call last):
  File "/home/u1/.config/blender/2.90/scripts/addons/YABEE/yabee_libs/egg_writer.py", line 1600, in write_out
    materials_str, USED_MATERIALS, USED_TEXTURES = get_egg_materials_str(selected_obj)
  File "/home/u1/.config/blender/2.90/scripts/addons/YABEE/yabee_libs/egg_writer.py", line 1174, in get_egg_materials_str
    for node in bpy.data.materials[0].node_tree.nodes:
AttributeError: 'NoneType' object has no attribute 'nodes'
```

**Comment by jimmytheguitarman** [comment]
In have the same issue, it is possible to load the .egg in panda3d. But without color.

**Comment by kergalym** [comment]
Reproduced with default cube in Blender 2.83, but not in 2.80. Something is changed, let me fix it.

**Comment by kergalym** [comment]
Hi, 
Bug is related to Dots Strokes material which is a used by several default brushes of Grease Pencil. You can delete it in Blender File of the Outliner before exporting your scene.

<img src="https://i.imgur.com/IDcauZ4.jpg" />

[Confirmed in Blender bugtracker](https://developer.blender.org/T75888).

**Comment by kergalym** [comment]
Fixed by [58a229f6fb1fda915c515a1eeb57e61dd43b16b2](https://github.com/kergalym/YABEE/commit/58a229f6fb1fda915c515a1eeb57e61dd43b16b2)
